madlibs.js

```diff
@@ -22,21 +22,151 @@
  *  { word: "store", pos: "noun" }
  *  { word: "," }
  *  ....
- * 
+ *
  * There are multiple ways to do this, but you may want to use regular expressions.
  * Please go through this lesson: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/regular-expressions/
  */
+
 function parseStory(rawStory) {
-  // Your code here.
-  return {}; // This line is currently wrong :)
+  const dot = '.';
+  const comma = ',';
+  const noun = '[n]';
+  const verb = '[v]';
+  const adjective = '[a]';
+  rawStory = rawStory.replace(dot, ' .');
+  rawStory = rawStory.replace(comma, ' , ');
```

Comment:
This looks good but you can also chain replace functions like this
```suggestion
  rawStory = rawStory.replace(dot, ' .').replace(comma, ' , ');
```